### PR TITLE
Add fr translation

### PR DESCRIPTION
**Pull request by jimbiscuit**

### CODE REVIEW

File: src/plone/restapi/locales/fr/LC_MESSAGES/plone.restapi.po [hunk 1]
```diff
@@ -0,0 +1,273 @@
...
+
+#. Default: "Root node"
+#: plone/restapi/services/contextnavigation/get.py:57
+msgid "label_navigation_root_path"
+msgstr "Noeud principal"
+
+#. Default: "Start level"
+#: plone/restapi/services/contextnavigation/get.py:95
+msgid "label_navigation_startlevel"
+msgstr "'Niveau de démarrage"
```
Comment (mpeeters) [inline src/plone/restapi/locales/fr/LC_MESSAGES/plone.restapi.po:239]: extra `'`

File: src/plone/restapi/locales/fr/LC_MESSAGES/plone.restapi.po [hunk 2]
```diff
@@ -0,0 +1,273 @@
...
+#: plone/restapi/services/contextnavigation/get.py:58
+msgid "help_navigation_root"
+msgstr "Vous pouvez rechercher et choisir un dossier qui servira de racine à l'arborescence de navigation. Laissez vide pour utiliser la racine du site Plone."
+
+#. Default: "An integer value that specifies the number of folder levels below the site root that must be exceeded before the navigation tree will display. 0 means that the navigation tree should be displayed everywhere including pages in the root of the site. 1 means the tree only shows up inside folders located in the root and downwards, never showing at the top level."
+#: plone/restapi/services/contextnavigation/get.py:96
+msgid "help_navigation_start_level"
+msgstr "Une valeur entière qui spécifie le nombre de niveaux de dossier sous la racine du site qui doit être dépassé avant que l'arborescence de navigation s'affiche. 0 signifie que l'arborescence de navigation doit être affichée partout, y compris les pages à la racine du site. 1 signifie que l'arborescence n'apparaît qu'à l'intérieur des dossiers situés à la racine et vers le bas, sans jamais s'afficher au niveau supérieur."
```
Comment (mpeeters) [inline src/plone/restapi/locales/fr/LC_MESSAGES/plone.restapi.po:209]: `Une valeur entière` -> `un nombre entier`

Comment (mpeeters) [reply src/plone/restapi/locales/fr/LC_MESSAGES/plone.restapi.po:209]: `y compris` -> `en ce compris`

Comment (mpeeters) [reply src/plone/restapi/locales/fr/LC_MESSAGES/plone.restapi.po:209]: `et vers le bas` -> `et en dessous`

File: src/plone/restapi/locales/fr/LC_MESSAGES/plone.restapi.po [hunk 3]
```diff
@@ -0,0 +1,273 @@
...
+
+#. Default: "If selected, the navigation tree will only show the current folder and its children at all times."
+#: plone/restapi/services/contextnavigation/get.py:84
+msgid "help_current_folder_only"
+msgstr "Si cette option est sélectionnée, l'arborescence de navigation n'affichera à tout moment que le dossier actuel et ses enfants."
+
+#. Default: "Whether or not to show the top, or 'root', node in the navigation tree. This is affected by the 'Start level' setting."
+#: plone/restapi/services/contextnavigation/get.py:69
+msgid "help_include_top_node"
+msgstr "Afficher ou non le nœud supérieur ou "racine" dans l'arborescence de navigation. Ceci est affecté par le paramètre 'Niveau de démarrage'."
```
Comment (mpeeters) [inline src/plone/restapi/locales/fr/LC_MESSAGES/plone.restapi.po:199]: `"` must be `'`

File: src/plone/restapi/locales/fr/LC_MESSAGES/plone.restapi.po [hunk 4]
```diff
@@ -0,0 +1,273 @@
...
+msgid "If enabled, the portlet will not show thumbs."
+msgstr "Si activé, le portlet n'affichera pas les miniatures."
+
+#: plone/restapi/services/users/add.py:296
+msgid "If you pass 'old_password' you have to pass 'new_password'"
+msgstr "Si vous passez 'ancien mot de passe' vous devez passer 'nouveau mot de passe'"
+
+#: plone/restapi/services/users/add.py:290
+msgid "If you pass 'reset_token' you have to pass 'new_password'"
+msgstr "Si vous passez "réinitialiser le jeton", vous devez passer "nouveau mot de passe""
```
Comment (mpeeters) [inline src/plone/restapi/locales/fr/LC_MESSAGES/plone.restapi.po:88]: `"` must be `'`

File: src/plone/restapi/locales/fr/LC_MESSAGES/plone.restapi.po [hunk 5]
```diff
@@ -0,0 +1,273 @@
...
+msgid "Enables Volto Blocks support"
+msgstr "Active la prise en charge des blocs Volto"
+
+#: plone/restapi/configure.zcml:81
+msgid "Enables blocks on the Document content type"
+msgstr "Active les blocs sur le type de contenu Document"
+
+#: plone/restapi/services/contextnavigation/get.py:133
+msgid "Enter a valid scale name (see 'Image Handling' control panel) to override (e.g. icon, tile, thumb, mini, preview, ... ). Leave empty to use default (see 'Site' control panel)."
+msgstr "Saisissez un nom valide d'échelle (cf. « Gestion des images » dans le panneau de configuration) pour outrepasser (p. ex. icône, tuile, miniature, mini, prévisualisation, etc.). Laisser vide pour utiliser les échelles par défaut (cf. « Site » dans le panneau de configuration)."
```
Comment (mpeeters) [inline src/plone/restapi/locales/fr/LC_MESSAGES/plone.restapi.po:63]: scale names should remain in english (icon, tile, thumb, ...)

File: src/plone/restapi/locales/fr/LC_MESSAGES/plone.restapi.po [hunk 6]
```diff
@@ -0,0 +1,273 @@
...
+msgid "Adds sample content types for testing"
+msgstr "Ajoute des exemples de types de contenu pour les tests"
+
+#: plone/restapi/configure.zcml:65
+msgid "Adds sample workflows for testing"
+msgstr "Ajoute des exemples processus pour les tests"
+
+#: plone/restapi/services/aliases/add.py:106
+msgid "Alternative urls that point to themselves will cause an endless cycle of redirects."
+msgstr "Les URL alternatives qui pointent vers elles-mêmes provoqueront un cycle sans fin de redirections."
```
Comment (mpeeters) [inline src/plone/restapi/locales/fr/LC_MESSAGES/plone.restapi.po:39]: `URL` -> `URLs`

File: src/plone/restapi/locales/fr/LC_MESSAGES/plone.restapi.po [hunk 7]
```diff
@@ -0,0 +1,273 @@
...
+msgid "${sender_fullname} via ${portal_title}"
+msgstr "${sender_fullname} via ${portal_title}"
+
+#: plone/restapi/services/email_send/post.py:74
+msgid "A portal user via ${portal_title}"
+msgstr "Un portail utilsateur via ${portal_title}"
+
+#: plone/restapi/configure.zcml:73
+msgid "Adds sample content for performance testing"
+msgstr "Ajoute un exemple de contenu pour les tests de performances"
```
Comment (mpeeters) [inline src/plone/restapi/locales/fr/LC_MESSAGES/plone.restapi.po:27]: `des exemples de contenus` ?

File: src/plone/restapi/locales/fr/LC_MESSAGES/plone.restapi.po [hunk 8]
```diff
@@ -0,0 +1,273 @@
...
+"Preferred-Encodings: utf-8 latin1\n"
+"Domain: plone.restapi\n"
+
+#: plone/restapi/services/email_send/post.py:81
+msgid "${sender_fullname} via ${portal_title}"
+msgstr "${sender_fullname} via ${portal_title}"
+
+#: plone/restapi/services/email_send/post.py:74
+msgid "A portal user via ${portal_title}"
+msgstr "Un portail utilsateur via ${portal_title}"
```
Comment (mpeeters) [inline src/plone/restapi/locales/fr/LC_MESSAGES/plone.restapi.po:23]: typo and `portail utilisateur` does not mean anything

File: src/plone/restapi/locales/fr/LC_MESSAGES/plone.restapi.po (hunk 2 above)
Comment (jimbiscuit) [reply src/plone/restapi/locales/fr/LC_MESSAGES/plone.restapi.po]: Fix

File: src/plone/restapi/locales/fr/LC_MESSAGES/plone.restapi.po (hunk 3 above)
Comment (jimbiscuit) [reply src/plone/restapi/locales/fr/LC_MESSAGES/plone.restapi.po]: Fix

File: src/plone/restapi/locales/fr/LC_MESSAGES/plone.restapi.po (hunk 4 above)
Comment (jimbiscuit) [reply src/plone/restapi/locales/fr/LC_MESSAGES/plone.restapi.po]: Fix

File: src/plone/restapi/locales/fr/LC_MESSAGES/plone.restapi.po (hunk 5 above)
Comment (jimbiscuit) [reply src/plone/restapi/locales/fr/LC_MESSAGES/plone.restapi.po]: Fix

File: src/plone/restapi/locales/fr/LC_MESSAGES/plone.restapi.po (hunk 6 above)
Comment (jimbiscuit) [reply src/plone/restapi/locales/fr/LC_MESSAGES/plone.restapi.po]: Fix

File: src/plone/restapi/locales/fr/LC_MESSAGES/plone.restapi.po (hunk 7 above)
Comment (jimbiscuit) [reply src/plone/restapi/locales/fr/LC_MESSAGES/plone.restapi.po:27]: Fix

File: src/plone/restapi/locales/fr/LC_MESSAGES/plone.restapi.po (hunk 8 above)
Comment (jimbiscuit) [reply src/plone/restapi/locales/fr/LC_MESSAGES/plone.restapi.po]: Fix

File: src/plone/restapi/locales/fr/LC_MESSAGES/plone.restapi.po [hunk 9]
```diff
@@ -0,0 +1,273 @@
...
+msgid "You can't set a password without a password reset token."
+msgstr "Vous ne pouvez pas définir de mot de passe sans un jeton de réinitialisation de mot de passe."
+
+#: plone/restapi/services/users/update.py:125
+msgid "You can't update the properties of this user"
+msgstr "Vous ne pouvez pas mettre à jour les propriétés de cet utilisateur"
+
+#: plone/restapi/services/users/add.py:284
+msgid "You can't use 'reset_token' and 'old_password' together."
+msgstr "Vous ne pouvez pas utiliser 'jeton de réinitialisation' et 'ancien mot de passe' ensemble."
```
Comment (mpeeters) [inline src/plone/restapi/locales/fr/LC_MESSAGES/plone.restapi.po:172]: `reset_token` and `old_password` must remains in english

**Review by erral — APPROVED** [review]

**Review by mpeeters — APPROVED** [review]

**Pull request merged**